<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkRenderWindowDataStorageInspector</class>
 <widget class="QWidget" name="QmitkRenderWindowDataStorageInspector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>246</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Render window manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Render window overview</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeView" name="renderWindowTreeView"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QRadioButton" name="radioButtonAxial">
       <property name="text">
        <string>Axial</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="radioButtonCoronal">
       <property name="text">
        <string>Coronal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QRadioButton" name="radioButtonSagittal">
       <property name="text">
        <string>Sagittal</string>
       </property>
      </widget>
     </item>
	 <item row="0" column="3">
      <widget class="QRadioButton" name="radioButton3D">
       <property name="text">
        <string>3D</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
